### ISSUE TITLE
How do we want to represent asides?

### ISSUE THREAD

**Issue by boldingd** [typedrat/renaissance#1]
I need some kind of aside block (and a block for fiction inserts would be nice).
So, one possible answer to this question is, "just insert a raw HTML element, like `<aside>`, because markdown isn't supposed to be a publishing format."
But, _apparently,_ some people consider not-pure-markdown files a bad thing.
So, here's the question: do we repurpose some block-level element(s) that Markdown supports (like the blockquote), or do we just insert some HTML manually (like `<aside>`)?

**Comment by crutchcorn** [comment]
I'm not @typedrat, but I'll see if I can get some movement/insight on this issue based on a discussion I had with her about the project's architecture:

Pure markdown files would be nice for the site as she's going to be using some MD->PDF magics to be able to make the site able to export perfectly to a PDF as well (and adding HTML breaks a lot of this). As a result, you can do something like `blockquote` as you'd mentioned (what I've personally ended up doing for my blog, which I think, when styled properly, can look nice)
![image](https://user-images.githubusercontent.com/9100169/62953394-4f0b4100-bda2-11e9-8bae-781c2ee882c3.png)

**Comment by boldingd** [comment]
I've got two concerns with that:
1) I had in mind doing multiple _types_ of asides, with different formatting--like a different background for "gm guidance" versus "design notes" versus "example text".
2) I was also hoping to eventually add bits of short fiction as fluff text; how do we set those apart?

... and well 3) I've already added a bunch of \<asside\>s but it'll be easy enough to find those.

I think there are markdown flavors that are extensible, and I think there are markdown-like languages (like reStructuredText) that are designed to be extensible; should we consider switching to one of those?  Would that require @typedrat to rewrite the whole static site generator?
